js/components/checkbox.js

```diff
@@ -61,12 +57,11 @@ Fliplet.FormBuilder.field('checkbox', {
     }
   },
   created: function () {
-    if (!!this.defaultValue) {
-      this.value = this.defaultValue.split(/\n/);
-      this.updateValue(this.name, this.value);
-    } else if (!Array.isArray(this.value)) {
-      this.value = [];
-      this.updateValue(this.name, this.value);
+    // We have removed defaultValue variable and set value intead
```

Comment:
I don't understand why the comment is mentioning something that should be mentioned in the PR description.

There shouldn't be a comment here as the code is straight forward